javascript/code-challenges/challenges-01.test.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named `addOne` that takes an array of numbers, and returns a new array of the numbers, incremented by 1.
+
+Use `forEach` to loop over the input array and work with each value.  Push the new value into a local array. Return the local array;
+------------------------------------------------------------------------------------------------ */
+
+const addOne = (arr) => {
+  let arr2 = [];
+  arr.forEach(element => {
+    let newNum = element + 1;
+    arr2.push(newNum);
+  });
```

Comment:
Good job on starting the challenges 👍 

This one is almost there, we just need to take a look at which `arr` we are using forEach on. Do we want to use forEach on arr2 or perhaps the input arr?